### ISSUE TITLE
some values ​​are not saved/applied

### ISSUE THREAD

**Issue by MrAnon-dev** [d3ward/nextntp#20]
Hi,

I have installed this extension in kiwi browser and the following appearance values ​​have no effect

Tiles grid:

- Folder Tile Background
- Folder Label Background
- Folder Background Icon

Enviroment:
```
Android 11 Realme UI 2
NextNTP v1.0.4 (chrome://newtab/)
Kiwi Browser v96.0.4664.46
```

Thank you so much

regards

**Comment by vjrmy** [comment]
Yes, Me too. I can add new tile but i can't erase the tile or edit it

**Comment by d3ward** [comment]
Fixed with v2.0.0
